bsconfig.schema.json

```
@@ -100,7 +100,12 @@
             "default": false
         },
         "copyToStaging": {
-            "description": "If true, the files are copied to staging. This setting is ignored when deploy is enabled or if createPackage is enabled",
+            "description": "(Deprecated use 'noEmit') If true, the files are copied to staging. This setting is ignored when deploy is enabled or if createPackage is enabled",
```

```suggestion
            "description": If true, the files are copied to staging. This setting is ignored when deploy is enabled or if createPackage is enabled",
                "deprecationMessage": "Deprecated. Use `noEmit` instead.",
```

Should use `deprecationMessage` for the rest of these below as well